server/src/routes/admin.ts

```diff
@@ -18,14 +18,14 @@ adminRouter.post("/signup", async (req: Request, res: Response) => {
       await prisma.$disconnect();
       return res.status(403).json({ message: "Admin email already exists" });
     }
-
     const hashedPassword: string = await bcrypt.hash(password, 8);
     await prisma.admin.create({
       data: {
         email,
         hashedPassword: hashedPassword,
       },
-    });
+    })
+
```

Comment:
Why you need space here?
Just run "npm run format" before commit/push anything. Prettier will take care of it.